### PR TITLE
Update Readme to add detailed information and update the CR example

### PR DESCRIPTION
**Pull request by DTMad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,14 +41,19 @@ $ kubectl apply -f https://github.com/Dynatrace/dynatrace-operator/releases/late
...
 assistance please refere
-to [Create user-generated access tokens](https://www.dynatrace.com/support/help/get-started/introduction/why-do-i-need-an-access-token-and-an-environment-id/#create-user-generated-access-tokens)
-.
+to [Create user-generated access tokens.](https://www.dynatrace.com/support/help/get-started/introduction/why-do-i-need-an-access-token-and-an-environment-id/#create-user-generated-access-tokens)
+
+Make sure the *Dynatrace API* token has the following permission:
+
+* Access problem and event feed, metrics and topology
+* Read configuration
+* Write configuration
```
Comment (baichinger) [inline README.md:50]: Only necessary when deployed via `install.sh`. Would rather provide a separate README next to the script.

File: README.md [hunk 2]
```diff
@@ -90,6 +111,31 @@ at [our DynaKube Custom Resource example](https://raw.githubusercontent.com/Dyna
 $ kubectl apply -f cr.yaml
 ```
 
+#### Enable Kubernetes API monitoring
+
+To get native Kubernetes metrics, you need to connect the Kubernetes API to Dynatrace.
+
+1.Get the Kubernetes API URL
```
Comment (baichinger) [inline README.md:118]: An extra space between the number and text would be nice.

File: README.md [hunk 3]
```diff
@@ -118,14 +164,19 @@ $ oc apply -f https://github.com/Dynatrace/dynatrace-operator/releases/latest/do
...
 assistance please refere
-to [Create user-generated access tokens](https://www.dynatrace.com/support/help/get-started/introduction/why-do-i-need-an-access-token-and-an-environment-id/#create-user-generated-access-tokens)
-.
+to [Create user-generated access tokens.](https://www.dynatrace.com/support/help/get-started/introduction/why-do-i-need-an-access-token-and-an-environment-id/#create-user-generated-access-tokens)
+
+Make sure the *Dynatrace API* token has the following permission:
+
+* Access problem and event feed, metrics and topology
+* Read configuration
+* Write configuration
```
Comment (baichinger) [inline README.md:173]: Same argument as above.

File: README.md (hunk 1 above)
Comment (DTMad) [reply README.md]: Removed from here

File: README.md (hunk 2 above)
Comment (DTMad) [reply README.md]: Added a space

File: README.md (hunk 3 above)
Comment (DTMad) [reply README.md]: Added a space

**Review by lrgar — APPROVED** [review]

**Pull request merged**